### PR TITLE
Add `Queryable` component to show a green add circle icon for the queryable UI elements

### PR DESCRIPTION
**Pull request by mertyildiran**
https://user-images.githubusercontent.com/2502080/143855611-1100b0a0-8d73-474f-8a23-9ec82b7b5472.mp4

Screenshot from 2021-11-29 14-38-51

### CODE REVIEW

**Comment by lirazyehezkel** [comment]
@mertyildiran The tooltip looks a bit messy IMO, I would remove it and use only the + icon. In addition, I would put a small delay (300ms for example) before showing the tooltip on element hover

**Comment by mertyildiran** [comment]
Updated the screenshots in cheatsheet.

File: ui/src/components/EntryDetailed/EntrySections.tsx [hunk 1]
```diff
@@ -15,23 +16,29 @@ interface EntryViewLineProps {
 }
 
 const EntryViewLine: React.FC<EntryViewLineProps> = ({label, value, updateQuery, selector, overrideQueryValue}) => {
+    var query = "";
```
Comment (lirazyehezkel) [inline ui/src/components/EntryDetailed/EntrySections.tsx:19]: change to let

File: ui/src/components/UI/Queryable.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    useEffect(() => {
+        let timer;
+        if (showAddedNotification) {
+            updateQuery(query);
+            timer = setTimeout(() => {
+                setAdded(false);
+            }, 1000);
+        }
+        return () => clearTimeout(timer);
+        // eslint-disable-next-line
```
Comment (lirazyehezkel) [inline ui/src/components/UI/Queryable.tsx]: I think if you will add updateQuery to the deps array and check in the top of the useEffect - if(!updateQuery) return;
you can avoid this eslint warning

**Review by lirazyehezkel — APPROVED** [review]

**Pull request merged**